snake/main.py

```diff
@@ -189,6 +193,11 @@ def checkered_surface(screen: pygame.Surface) -> pygame.Surface:
 
     return checkered
 
+def score_update(score):
+    font = pygame.font.SysFont("sans-sarif", 35, True)
```

Comment:
I don't think sans-sarif is a real font lol
Try pygame.font.get_fonts() to see the available fonts in your system